### DISCUSSION TITLE
LoadGltfBinary does not work

### DISCUSSION THREAD

**Discussion by Djokix98** [atteneder/glTFast#411] [Q&A] [answered]
Hello! I created an empty project, installed the plugin and tried to load the local gltf file with the LoadGltfBinary() method but it seems it does not work. There is also a helping "Uri" parameter in the method but it also doesn't seem to help. Any suggestions how to make this work?
Thanks in advance.

```
var filePath = Application.dataPath + "/TestModels/skull_salazar/scene.gltf";
byte[] data = File.ReadAllBytes(filePath);
GltfImport gltf = new GltfImport();
bool success = await gltf.LoadGltfBinary(data); // Does not work
// bool success = await gltf.LoadGltfBinary(data, new Uri(filePath)); // Does not work
// bool success = await gltf.Load(filePath); // Works

  if (success)
  {
      gltf.InstantiateMainScene(transform);
  }
  else
  {
      throw new Exception("MEGA ERROR");
  }
```
 
[GltfTesting.zip](https://github.com/atteneder/glTFast/files/8814514/GltfTesting.zip)

**Comment by atteneder** [comment, marked as answer]
If you want to know why it failed, use a logger:

```csharp
GltfImport gltf = new GltfImport(logger:new ConsoleLogger());
```

It'll tell you `Not a glTF-binary file`. `LoadGltfBinary` only loads glTF-Binary. A variant for JSON based glTFs is in the works (#409).
